### ISSUE TITLE
Large repository

### ISSUE THREAD

**Issue by effigies** [nipype/pydra#419]
```Bash
$ time git clone https://github.com/nipype/pydra.git
Cloning into 'pydra'...
remote: Enumerating objects: 7556, done.
remote: Counting objects: 100% (7556/7556), done.
remote: Compressing objects: 100% (933/933), done.
remote: Total 58509 (delta 6926), reused 7183 (delta 6623), pack-reused 50953
Receiving objects: 100% (58509/58509), 244.93 MiB | 3.16 MiB/s, done.
Resolving deltas: 100% (52389/52389), done.
git clone https://github.com/nipype/pydra.git  44.05s user 10.76s system 60% cpu 1:31.08 total
$ du -sh pydra && du -sh pydra/*/
259M	pydra
260K	pydra/docs/
11M	pydra/pydra/
8.0K	pydra/tutorial/
```

For a pretty lean working tree (10/11MB are a single test file) we have a very heavy git history. It might be worth doing some rewriting.

**Comment by djarecka** [comment]
never paid attention, but you're right... not sure, what I put it there... I used to have tutorial here, and this might be partially responsible

**Comment by effigies** [comment]
Realizing it might be gh-pages. Any objection to me squashing the entire history of `gh-pages` into a single commit to see if that helps reduce the repo size? If so we can just do it periodically or update the GH action to do it for us.

**Comment by effigies** [comment]
Actually I think `gh-pages` definitely is it:

```Shell
$ time git clone --single-branch https://github.com/nipype/pydra.git
Cloning into 'pydra'...
remote: Enumerating objects: 151, done.
remote: Counting objects: 100% (151/151), done.
remote: Compressing objects: 100% (107/107), done.
remote: Total 8122 (delta 82), reused 90 (delta 44), pack-reused 7971
Receiving objects: 100% (8122/8122), 12.83 MiB | 25.72 MiB/s, done.
Resolving deltas: 100% (6184/6184), done.
git clone --single-branch https://github.com/nipype/pydra.git  0.81s user 0.29s system 83% cpu 1.318 total
$ du -sh pydra
25M	pydra
```

**Comment by effigies** [comment]
Squashed.

```Shell
$ time git clone https://github.com/nipype/pydra.git 
Cloning into 'pydra'...
remote: Enumerating objects: 1566, done.
remote: Counting objects: 100% (1566/1566), done.
remote: Compressing objects: 100% (262/262), done.
remote: Total 9623 (delta 1361), reused 1472 (delta 1304), pack-reused 8057
Receiving objects: 100% (9623/9623), 18.88 MiB | 8.23 MiB/s, done.
Resolving deltas: 100% (7477/7477), done.
git clone https://github.com/nipype/pydra.git  1.51s user 0.41s system 48% cpu 3.985 total
$ du -sh pydra 
31M	pydra
```

**Comment by effigies** [comment]
Should be resolved as of 7bf22ff.
